doc/src/Errors_messages.txt

```diff
@@ -10187,8 +10181,6 @@ Self-explanatory. :dd
 
 {Unrecognized virial argument in pair_style command} :dt
```

Comment:
this line needs to be removed as it is the header for the ':dd' section that you removed.

Reply:
thanks.